### ISSUE TITLE
debugger bottom window constantly resizes

### ISSUE THREAD

**Issue by ta946** [daveleroy/sublime_debugger#153]
im using v0.6.2 and on windows, the bottom window that has the call stack and variables is constantly changing its internal width. So you see a horizontal scroll bar, then it disappears, then appears again.

this is printed in the console
```
overscan 1.400000000000091: adjusting rem_width: 0.735
reloading settings Packages/User/debugger.sublime-settings
underscan -9.0: adjusting rem_width: 0.73
reloading settings Packages/User/debugger.sublime-settings
overscan 1.400000000000091: adjusting rem_width: 0.735
reloading settings Packages/User/debugger.sublime-settings
underscan -9.0: adjusting rem_width: 0.73
reloading settings Packages/User/debugger.sublime-settings
```

It constantly does that when I start the debugger and that window appears. resizing sublime to a smaller width sometimes stops that behaviour, other times it slowly increases the width 1 step at a time till it stops

```
underscan -63.40000000000009: adjusting rem_width: 0.57
reloading settings Packages/User/debugger.sublime-settings
underscan -57.0: adjusting rem_width: 0.605
reloading settings Packages/User/debugger.sublime-settings
underscan -50.60000000000002: adjusting rem_width: 0.635
reloading settings Packages/User/debugger.sublime-settings
underscan -41.799999999999955: adjusting rem_width: 0.665
reloading settings Packages/User/debugger.sublime-settings
underscan -29.799999999999955: adjusting rem_width: 0.69
reloading settings Packages/User/debugger.sublime-settings
underscan -21.0: adjusting rem_width: 0.705
reloading settings Packages/User/debugger.sublime-settings
underscan -14.600000000000023: adjusting rem_width: 0.72
reloading settings Packages/User/debugger.sublime-settings
underscan -5.7999999999999545: adjusting rem_width: 0.73
reloading settings Packages/User/debugger.sublime-settings
```

**Comment by daveleroy** [comment]
You should try 0.6.3 when its on package control I adjusted the calculation so it should be a little more precise. If all else fails you can set `ui_rem_width_scale_adjust_automatically` to `false` and choose a value manually.

**Comment by ta946** [comment]
solved with 0.6.3. works fast and flawlessly
